Apptainer image build failing in CI

**Describe the bug**
Since July 1, 2026, the Apptainer image build is failing in CI.

**To Reproduce**
Steps to reproduce the behavior:
1. Open de development branch in push into it.
2. Contemplate the CI failing.

**Expected behavior**
The Apptainer image build should succeed !

**Supporting information**
For instance, have a look at: https://github.com/cta-observatory/nectarchain/actions/runs/28506425750/job/84548497833

**Additional information**
It seems the image build fails when embedding environment variables in the `conda` environment:

```shell
+ conda env config vars set X509_CERT_DIR=/opt/conda/envs/nectarchain/etc/grid-security/certificates X509_VOMS_DIR=/opt/conda/envs/nectarchain/etc/grid-security/vomsdir X509_VOMSES=/opt/conda/envs/nectarchain/etc/grid-security/vomses
+ local cmd=env
+ __conda_exe env config vars set X509_CERT_DIR=/opt/conda/envs/nectarchain/etc/grid-security/certificates X509_VOMS_DIR=/opt/conda/envs/nectarchain/etc/grid-security/vomsdir X509_VOMSES=/opt/conda/envs/nectarchain/etc/grid-security/vomses
+ [ -n  ]
+ /opt/conda/bin/conda env config vars set X509_CERT_DIR=/opt/conda/envs/nectarchain/etc/grid-security/certificates X509_VOMS_DIR=/opt/conda/envs/nectarchain/etc/grid-security/vomsdir X509_VOMSES=/opt/conda/envs/nectarchain/etc/grid-security/vomses
Traceback (most recent call last):
  File "/opt/conda/bin/conda", line 12, in <module>
    from conda.cli import main
ModuleNotFoundError: No module named 'conda'
FATAL:   While performing build: while running engine: while running %post section: exit status 1
Error: Process completed with exit code 255.
```

--- a/Singularity
+++ b/Singularity
@@ -56,7 +56,7 @@ From: condaforge/miniforge3
 
     # Optionally install and configure DIRAC:
     mamba install --quiet -y -c conda-forge "dirac-grid=8"
-    conda env config vars set X509_CERT_DIR=${CONDA_PREFIX}/etc/grid-security/certificates X509_VOMS_DIR=${CONDA_PREFIX}/etc/grid-security/vomsdir X509_VOMSES=${CONDA_PREFIX}/etc/grid-security/vomses
+    mamba env config vars set X509_CERT_DIR=${CONDA_PREFIX}/etc/grid-security/certificates X509_VOMS_DIR=${CONDA_PREFIX}/etc/grid-security/vomsdir X509_VOMSES=${CONDA_PREFIX}/etc/grid-security/vomses
     mamba deactivate
     mamba activate nectarchain
     pip install "CTADIRAC<3"

